src/components/PlaylistPage/PlaylistPage.tsx

```diff
@@ -23,32 +24,44 @@ const PlaylistPage = () => {
     const { id } = useParams<{ id: string }>();
     const currentPlaylist = useSelector(selectCurrentPlaylist);
     const currentUser = useSelector(selectCurrentUser);
-    const [addSongActive, setAddSongActive] = useState(false);
-    const [inviteActive, setInviteActive] = useState(false);
-    const [playerActive, setPlayerActive] = useState(false);
-    const [confirmationIsVisible, setConfirmationIsVisible] = useState(false)
+    const [addSongActive, setAddSongActive] = useState<boolean>(false);
+    const [inviteActive, setInviteActive] = useState<boolean>(false);
+    const [playerActive, setPlayerActive] = useState<boolean>(false);
+    const [confirmationIsVisible, setConfirmationIsVisible] = useState<boolean>(false)
```

Comment:
if there is an initial value, the typescipt will figure it out that this is a boolean type

Reply:
fixed